src/plugins/sails.js

```diff
@@ -168,8 +168,12 @@ io.socket.on('gameCreated', function (evData) {
   store.commit('addGameToList', newGame);
 });
 
-io.socket.on('gameStarting', function (evData) {
-  store.commit('removeGame', evData);
+io.socket.on('gameStarting', function ({ gameId }) {
```

Comment:
Should normalize the tense here. Events make more sense in past tense imo so `gameStarted`